app/client/src/sagas/ActionExecution/PluginActionSaga/baseExectutePluginSaga.ts

```
@@ -0,0 +1,689 @@
+        // TODO: Fix this the next time the file is edited
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const newVal: Record<string, any> = yield call(
+          resolvingBlobUrls,
+          val,
+          executeActionRequest,
+          i,
+          true,
+          arrDatatype,
+        );
+
+        if (newVal.hasOwnProperty("blobUrlPaths")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)